multiboot2-header/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "multiboot2-header"
+description = """
+Library with type definitions and parsing functions for Multiboot2 headers.
+This library is `no_std` and can be used in a bootloaders.
+"""
+# TODO keep in Sync with "multiboot2"
+version = "0.12.0"
```

Comment:
In my experience with the ACPI crates, this is not the best idea, as it confuses people into thinking the crate has changed when it has not, and doesn't really have any advantages imo. I think we should let them float at separate versions. This would also allow one to be stabilized separately from the other.

Reply:
Thanks, I wasn't sure about that! :)